Python/Product/Cookiecutter/ViewModel/CookiecutterViewModel.cs

```diff
@@ -273,9 +273,9 @@ class CookiecutterViewModel : INotifyPropertyChanged {
             _statusImageVisibility = image.HasValue && !progressVisible ? Visibility.Visible : Visibility.Collapsed;
             _spinningStatusImageVisibility = image.HasValue && progressVisible ? Visibility.Visible : Visibility.Collapsed;
             bool v = (visible ?? !string.IsNullOrEmpty(message));
-            _statusVisibility = v ? Visibility.Visible : Visibility.Collapsed;
-            _statusProgressVisibility = progressVisible ? Visibility.Visible : Visibility.Collapsed;
-            _statusUpdateProgressVisibility = updateProgressVisible ? Visibility.Visible : Visibility.Collapsed;
+            _statusVisibility = v ? Visibility.Visible : Visibility.Hidden;
```

Comment:
Those are fields. WPF won't be notified about updated values.

Reply:
The PropertyChanged notifications are just a few lines below.